src/sentry/models/orgauthtokenproject.py

```diff
@@ -0,0 +1,25 @@
+from sentry.db.models import (
+    BaseManager,
+    FlexibleForeignKey,
+    Model,
+    control_silo_only_model,
+    sane_repr,
+)
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@control_silo_only_model
+class OrgAuthTokenProject(Model):
```

Comment:
You should put this model in `orgauthtoken.py` since it belongs to the same cluster of models related to org auth tokens.

Reply:
after talking this through with @mitsuhiko , we decided to skip the projects for now!

Reply:
Due to the nature of the relationship, it's not a big deal at all for the db level.